src/settings.rs

```diff
@@ -19,6 +19,15 @@ use crate::{
     },
 };
 
+pub fn get_langs() -> HashMap<String, String>{
+    let mut langs = HashMap::new();
+    langs.insert("en".to_string(), "English".to_string());
+    langs.insert("zh-cn".to_string(), "Simplified Chinese".to_string());
+    langs.insert("zh-tw".to_string(), "Traditional Chinese".to_string());
+    langs.insert("ja".to_string(), "Japanese".to_string());
+    langs
+}
+
```

Comment:
I appreciate you supporting all of these languages from the beginning but avoid creating hashmaps when not required. We can list out the languages with an enum.



 ```suggestion
use std::string::ToString;
use strum_macros::Display;

#[derive(Display, Debug)]
#[strum(serialize_all = "kebab-case")]
enum Language {
    #[strum(to_string = "English")]
    En,
    #[strum(to_string = "Simplified Chinese")]
    ZhCn,
    #[strum(to_string = "Traditional Chinese")]
    ZhTw,
    #[strum(to_string = "Japanese")]
    Ja,
}

Then you can go from 

```rust
// "en" => Language::En
println!("{:?}", Language:from_str("zh-cn"));
// Language::En => "English"
println!("{}", Language::En.to_string());
```

Examples from the strum library:
https://github.com/Peternator7/strum/blob/master/strum_tests/tests/serialize_all.rs
https://github.com/Peternator7/strum/blob/master/strum_tests/tests/to_string.rs